indico/modules/events/registration/controllers/management/tickets.py

```diff
@@ -60,11 +60,10 @@ def _process(self):
         if form.validate_on_submit():
             form.populate_obj(self.regform)
             db.session.flush()
-            return redirect(url_for('.tickets', self.regform))
+            return jsonify_data()
 
-        return WPManageRegistration.render_template('management/regform_tickets.html', self.event,
-                                                    regform=self.regform, form=form,
-                                                    can_enable_tickets=self._check_ticket_app_enabled())
+        return jsonify_template('events/registration/management/regform_tickets.html', event=self.event,
```

Comment:
It's worth mentioning positional arguments in `WPWhatever.render_template` are not passed to the template, while keyword arguments are. If you don't need to use `{{ event }}` in the template there's no need to pass it as a keyword argument.